config/initializers/secure_headers.rb

```diff
@@ -9,7 +9,7 @@
 
   tta_service_hosts = []
   tta_service_hosts << URI.parse(ENV["TTA_SERVICE_URL"]).host if ENV["TTA_SERVICE_URL"].present?
-  google_analytics = %w[*.google-analytics.com *.googletagmanager.com *.googleusercontent.com *.gstatic.com s.ytimg.com *.googleadservices.com *.googleads.g.doubleclick.net]
+  google_analytics = %w[*.google-analytics.com *.googletagmanager.com *.googleusercontent.com *.gstatic.com s.ytimg.com *.googleadservices.com *.googleads.g.doubleclick.net https://googleads.g.doubleclick.net]
```

Comment:
should it have the `https://` here or should it just be `googleads.g.doubleclick.net`?

Reply:
I think it was in with the `https://` previously, though it probably works with both 🤷‍♂️